Library/Homebrew/test/lib/config.rb

```diff
@@ -12,7 +12,9 @@
 
 # Paths pointing into the Homebrew code base that persist across test runs
 HOMEBREW_LIBRARY_PATH  = Pathname.new(File.expand_path("../../..", __FILE__))
-HOMEBREW_ENV_PATH      = HOMEBREW_LIBRARY_PATH.parent+"ENV"
+HOMEBREW_ENV_PATH      = \
+  HOMEBREW_LIBRARY_PATH.parent+"Homebrew/env"
```

Comment:
Why does this line need to be wrapped, but the line directly underneath doesn't (though it has the exact same line length when unwrapped)? I think if we wanted to conserve some vertical space here, we'd be better off just dropping the unnecessary whitespace between the constant and the assignment operator.